### DISCUSSION TITLE
How to install - Win and Mac

### DISCUSSION THREAD

**Discussion by oisouole** [sorayuki/obs-multi-rtmp#121] [Q&A]
To install on Windows or Mac you gotta go to
[https://github.com/sorayuki/obs-multi-rtmp/releases/latest](https://github.com/sorayuki/obs-multi-rtmp/releases/latest)
and download the latest official release.

- **Windows users**: download .zip file
- **Mac users**: download .pkg file

**Comment by DharminderSuri** [comment]
downloaded as suggested for MAC. Now how to Integrate this pluggin with OBS running on my MAC

**Comment by robertsativa** [comment]
And what next? Where should we place it?

**Reply by CybaGirl** [reply]
Have you sorted this out yet?

Are you using the Portable or Full Version of OBS?

**Comment by iandouglas** [comment]
I haven't figured this out on Windows. I'm using the full version of OBS and not sure if that makes a difference for how to compile the code myself. I uploaded the released .exe file from GitHub to an online virus check and a dozen online scanners all said it contained a virus or malware of some sort, so I'm not running it on my local system at all until I can compile it myself.
